CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[UPDATE]** Improve & rename `TripSection` as `CardSection` to be able to handle `TripCard` and `QrCard` elements
```

Comment:
This is not an update, this is a breaking change 😉. 

If you want to avoid the breaking change, just add the new CardsSection component without removing the legacy `TripSection`. Then remove it in a future version when `CardsSection` will be used everywhere.

NB: you missed the **s** for CardsSection